core-services/template-deprecation/_allowlist.yaml

```diff
@@ -1789,6 +1789,9 @@ templates:
         pull-ci-openshift-sdn-release-4.8-e2e-aws-multitenant:
           generated: false
           kind: presubmit
+        pull-ci-openshift-vsphere-problem-detector-master-e2e-aws:
+          generated: true
+          kind: presubmit
```

Comment:
It doesn't look like this is necessary.

Reply:
It was generated by make new-repo. Shall I remove it?

Reply:
I read README and it indeed seems to be unnecessary. Removed.

Reply:
I read README and it indeed seems to be unnecessary. Removed.

Reply:
Hmm, I'll check that.